test/lib/ansible_test/_util/target/setup/bootstrap.sh

```diff
@@ -142,16 +142,28 @@ bootstrap_remote_freebsd()
 
     if [ "${controller}" ]; then
         jinja2_pkg="py${python_package_version}-jinja2"
-        cryptography_pkg="py${python_package_version}-cryptography"
         pyyaml_pkg="py${python_package_version}-yaml"
+        cryptography_pkg="py${python_package_version}-cryptography"
```

Comment:
Why was this line moved?

Reply:
because it is already included in 3.9 but not in other python versions

Reply:
why i changed it for the specific case but added back for the generic one